src/components/views/elements/AppTile.tsx

```diff
@@ -93,6 +93,8 @@ interface IProps {
     showLayoutButtons?: boolean;
     // Handle to manually notify the PersistedElement that it needs to move
     movePersistedElement?: MutableRefObject<(() => void) | undefined>;
+    // An element to render after the iframe as an overlay
+    overlay?: ReactNode;
```

Comment:
wouldn't it make more sense for this to be passed as `children`, rather than a regular prop?

Reply:
Children have different semantics of typically always being included, this only includes it when the iframe is shown, not when the loader or permissions prompt are shown

Reply:
> Children have different semantics of typically always being included

I'm not entirely convinced about that. And if it's true, doesn't the same logic apply to `PersistentApp`?

But I don't feel strongly about it.